Exercise_5.ipynb

```diff
@@ -0,0 +1,309 @@
+    "```\n",
+    "git checkout --track remote_name/branch_name\n",
+    "```\n",
+    "\n",
+    "Next, let's make a change in the \"updated_flyers\" branch.  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "attractive-shadow",
+   "metadata": {},
+   "source": [
+    "Make a change in your local repository.\n",
+    "Remember to do all modifications of the flyers directly via Jupyter Notebooks.\n",
+    "   * Click on *File* -> *Open* in the headers section of the notebook\n",
+    "   * Go to folder *work* and enter *party_planning*\n",
+    "   * Open *flyer_A*\n",
+    "   * Add more information to your flyer, i.e. music, dresscode, etc.\n",
+    "   \n",
+    "**Don't forget to save your modifications before coming back!**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "abstract-parker",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# add and commit your changes\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "hundred-greeting",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# check the status of your repository\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "weighted-humanitarian",
+   "metadata": {},
+   "source": [
+    "The output should look like this:\n",
+    "```\n",
+    "On branch updated_flyers\n",
+    "Your branch is ahead of 'test/updated_flyers' by 1 commit.\n",
+    "  (use \"git push\" to publish your local commits)\n",
+    "\n",
+    "nothing to commit, working tree clean\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "brown-guitar",
+   "metadata": {},
+   "source": [
+    "### Send information to remote repository\n",
+    "\n",
+    "Finally, we will send the changes we made to the remote repository.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "distant-spending",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# use \"git push remote_name branch_name\"  \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "historic-market",
+   "metadata": {},
+   "source": [
+    "The output should look something like this:\n",
+    "\n",
+    "```\n",
+    "Enumerating objects: 5, done.\n",
+    "Counting objects: 100% (5/5), done.\n",
+    "Delta compression using up to 8 threads\n",
+    "Compressing objects: 100% (3/3), done.\n",
+    "Writing objects: 100% (3/3), 306 bytes | 306.00 KiB/s, done.\n",
+    "Total 3 (delta 0), reused 0 (delta 0)\n",
+    "To ../party_planning_remote\n",
+    "   4f64811..a2244c4  updated_flyers -> updated_flyers\n",
+    "```"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Bash",
+   "language": "bash",
+   "name": "bash"
+  },
+  "language_info": {
+   "codemirror_mode": "shell",
+   "file_extension": ".sh",
+   "mimetype": "text/x-sh",
+   "name": "bash"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
I would add an additional part, where we examine another cycle if changing and pushing files in the remote for example. Could also be a bonus, so we have some time buffers for the fast people. 
Like that the perticipants see that Git-remotes are not one-way only, so the exchange is possible from both sides.